specifyweb/frontend/js_src/lib/localization/query.tsx

```diff
@@ -287,6 +287,14 @@ export const queryText = createDictionary({
     'en-us': 'Items have been returned',
     'ru-ru': 'Items have been returned',
   },
+  hideMappingEditor: {
```

Comment:
Do we already have these localization strings in the `workbench.tsx` file? If so, you can have query builder use those strings, rather than declaring them again here. (that is okay, because the mapping editor itself is also coming from the workbench files)